src/uct/ib/base/ib_md.c

```diff
@@ -1221,6 +1223,22 @@ static ucs_status_t uct_ib_query_md_resources(uct_component_t *component,
         num_resources++;
     }
 
+    if (num_resources != 0) {
+        status = ucs_sys_get_memlock_rlimit(&memlock_limit);
+        if ((status == UCS_OK) && (memlock_limit <= (500 * UCS_MBYTE))) {
+            /* Disable the RDMA devices because of too strict locked memory limit*/
+            ucs_diag("RDMA transports are disabled because max locked memory "
+                      "limit (%llu kbytes) is too low. Please set max locked "
+                      "memory (ulimit -l) to 'unlimited'",
```

Comment:
shall we write something like 
`"RDMA transports are disabled because max locked memory limit is too low (%llu kbytes). Please set ulimit -l to at least %llu kbytes or 'unlimited'", memlock_limit / UCS_KBYTE, ib_md->config.min_memlock_limit / UCS_KBYTE
`